src/ashell/comms-uart.c

```diff
@@ -72,7 +72,7 @@ const char *system_get_prompt()
     return system_prompt;
 }
 
-#define MAX_LINE_LEN 80
+#define MAX_LINE_LEN 90
```

Comment:
Maybe increase it to 120? I find even 90 characters per line is going to likely to run into issues down the road.